app/models/user.ts

```diff
@@ -5,6 +5,15 @@ export class User {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Column('varchar')
-  username: string;
+  @Column({ type: 'varchar' })
+  name: string;
+
+  @Column({ type: 'varchar' })
```

Comment:
remember that the tables in the database must be named using snake_case, for this, you can use a name property inside the column https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#33--database

Reply:
I know, but "name" does not contain two words, only one. I don't understand why I have to use snack_case here. Should I use "first_name" instead of "name"?

Reply:
this would be for last name

Reply:
this would be for last name